Tutorial MooseDoc and Images (#164)
idaholab/MALAMUTE: doc/content/tutorials/efas/introduction/concluding_remarks.md

Reviewer: Looks like the only issue remaining is debugging this pagination command. Work on debugging this by carefully checking for spelling errors and by comparing against the final page for other tutorials

This tutorial was created with the intention of providing initial instructions for MALAMUTE by exploring the relation between temperature and die wall thickness. After visualizing in Excel, the +Temperature vs Time+ data for each wall thickness, we see that the overall temperature decreases as the wall thickness increases. One exception to this pattern is the similar temperature values approximately halfway through the linear portion (approximately t = 1240 s to t = 1260 s) of the “cooling process”. Also, as the die wall thickness increases, the time at which max temperature occurs is higher. 

!media doc/content/media/tutorial_dcs5_images/tutorial_pngs/results_pngs/temp_vs_sim_time.png
       style=width:80%;margin-left:auto;margin-right:auto;
       id=temp_vs_sim_time
       caption=Temperature vs Time data for each of the five wall thicknesses

There is a sizable difference between each of the max temperature values for the various die wall thicknesses and a slight difference at the time which these max temperatures occur, as shown in [max_temp_vs_sim_time]. This occurrence might be explained by the electric field contour plots shown in [5mm_elec_field_and_temp]. 

> !media doc/content/media/tutorial_dcs5_images/tutorial_pngs/results_pngs/max_temp_vs_sim_time.png

Reviewer: ```suggestion
!media media/tutorial_dcs5_images/tutorial_pngs/results_pngs/max_temp_vs_sim_time.png
```

       style=width:100%;margin-left:auto;margin-right:auto;

>        id=max_temp_vs_sim_time

Reviewer: ```suggestion
       id=max-temp-vs-sim-time
```

The electric potential on the toolset steadily decreases from the top of the toolset to the bottom of it, as observed in the contour plot below. This indicates that the direction of electric current is from top to bottom, and this current direction corresponds with how the DCS-5 machine is constructed. 

!media media/tutorial_dcs5_images/tutorial_pngs/results_pngs/potential_temp_screenshot.png
    style=width:80%;margin-left:auto;margin-right:auto;

>     id=potential_temp_screenshot

Reviewer: update all the ids for the figures to use dashes instead of underlines
```suggestion
    id=potential-temp-screenshot
```

    caption=Electrical Potential and Temperature Contour Plots.

Electric field values slightly between different wall thickness, but noticeably changes on the graphite spacers. The electric field is highly concentrated in the center of these graphite spacers and seems to correspond with the higher temperature values in the same spot. This phenomenon is shown in [10mm_elec_field_and_temp], for example. However, the powder casing and die wall do not experience a similar spike in temperature or electric potential to the graphite spacers and may need to be considered when analyzing an EFAS toolset.

!media media/tutorial_dcs5_images/tutorial_pngs/results_pngs/5mm_elec_field_and_temp.png
    style=width:80%;margin-left:auto;margin-right:auto;
    id=5mm_elec_field_and_temp
    caption=Electric Field and Temperature Contour Plots for a 5 mm wall thickness at t = 954 seconds.

!media media/tutorial_dcs5_images/tutorial_pngs/results_pngs/10mm_elec_field_and_temp.png
    style=width:80%;margin-left:auto;margin-right:auto;
    id=5mm_elec_field_and_temp

>     caption=Electric Field and Temperature Contour Plots for a 10 mm wall thickness at t = 960 seconds.

Reviewer: Remove the `t =` in this caption and all others
```suggestion
    caption=Electric Field and Temperature Contour Plots for a 10 mm wall thickness at a simulation time of 960 seconds.
```

Reviewer: Make this change for all the captions that have an equal sign to remove that equal sign

!media media/tutorial_dcs5_images/tutorial_pngs/results_pngs/13.875mm_elec_field_and_temp.png
    style=width:80%;margin-left:auto;margin-right:auto;
    id=13.875mm-elec-field-and-temp
    caption=Electric Field and Temperature Contour Plots for a 13.875 mm wall thickness at t = 960 seconds.

!media media/tutorial_dcs5_images/tutorial_pngs/results_pngs/20mm-elec-field-and-temp.png
    style=width:80%;margin-left:auto;margin-right:auto;
    id=20mm-elec-field-and-temp
    caption=Electric Field and Temperature Contour Plots for a 20 mm wall thickness at t = 966 seconds.

> !media media/tutorial_dcs5_images/tutorial_pngs/results_pngs/25mm-elec-field-and-temp.png

Reviewer: try leaving this file path name with the underscores to match the file name you created in a previous commit:
```suggestion
!media media/tutorial_dcs5_images/tutorial_pngs/results_pngs/25mm_elec_field_and_temp.png
```
